backtrace-library/src/main/java/backtraceio/library/models/BacktraceExceptionHandler.java

```diff
@@ -80,4 +84,26 @@ private OnServerResponseEventListener getCallbackToDefaultHandler(final Thread t
             signal.countDown();
         };
     }
+
+    private List<BacktraceReport> transformExceptionIntoReports(Throwable throwable) {
+        final String exceptionTrace = UUID.randomUUID().toString();
+        BacktraceReport parent = null;
```

Comment:
You don't need whole BacktraceReport, you need String parentId

Reply:
The code was moved to another file. We're using string now